sale_order_import/wizard/sale_order_import.py

```diff
@@ -83,7 +81,11 @@ def _parse_file(self, filename, filecontent, detect_doc_type=False):
         }
         res = None
         if filetype and mimetype in supported_types["CSV"]:
-            res = False
+            res = (
+                self.parse_csv_order(filecontent, self.partner_id)
+                if self.partner_id
+                else False
+            )
```

Comment:
I have updated, may i add this code to fix the error?
![image](https://github.com/QuocDuong1306/edi/assets/127363167/2335a908-b9ed-4b3f-9c72-d1babda11ff9)

Reply:
definitely yes

Reply:
Could you review this PR again, is this ok..

Reply:
@QuocDuong1306 thanks for the update, if the CSV is not supported anymore, so we can remove the codeblocks / tests related to it, otherwise the test cases will fail